### PR TITLE
Feature dashboard front end completed. 

### PR DESCRIPTION
**Pull request by Rocket-Fish**
front end looks have been completed, 
style has been adjusted around slightly to better fit the mockup, 
refactored some code to make it more organized, 
added mobile responsiveness to sidebar
in regards to #15 #16 #17 #18 #19

### CODE REVIEW

File: client/src/components/chat-header/chat-header.component.jsx [hunk 1]
```diff
@@ -17,39 +31,81 @@ const ChatHeader = () => {
...
-    <Box boxShadow={1} py={3} px={5}>
-      <Grid container direction="row" justify="space-between" alignItems="center">
-        <Grid item>
-          <Grid container direction="row" justify="flex-start" alignItems="center" spacing={2}>
-            <Grid item>
-              <Typography>{title}</Typography>
+    <Box className={classes.header}>
+      <Box className={classes.bigPadd}>
+        <Grid container direction="row" justify="space-between" alignItems="center">
+          <Grid item xs sm>
```
Comment (bonnieli) [inline client/src/components/chat-header/chat-header.component.jsx]: the `xs sm` isn't assigned a value so i think it's just default as `false`. do we need these added props here?

File: client/src/components/chat-header/chat-header.component.jsx [hunk 2]
```diff
@@ -17,39 +31,81 @@ const ChatHeader = () => {
...
+
   return (
-    <Box boxShadow={1} py={3} px={5}>
-      <Grid container direction="row" justify="space-between" alignItems="center">
-        <Grid item>
-          <Grid container direction="row" justify="flex-start" alignItems="center" spacing={2}>
-            <Grid item>
-              <Typography>{title}</Typography>
+    <Box className={classes.header}>
+      <Box className={classes.bigPadd}>
```
Comment (bonnieli) [inline client/src/components/chat-header/chat-header.component.jsx]: a box in a box, can we add these 2 boxes inheriting both of those class properties together?

File: client/src/components/profile/profile.styles.jsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      backgroundColor: '#fff',
+    },
+  },
+  selected: {
+    backgroundColor: '#fff',
+  },
+  rounded: {
+    borderRadius: 16,
+  },
+  padddd: {
```
Comment (bonnieli) [inline client/src/components/profile/profile.styles.jsx]: be careful with naming haha whats `padddd` and i noticed you named something else `padd` as well

File: client/src/components/chat-header/chat-header.component.jsx (hunk 1 above)
Comment (Rocket-Fish) [reply client/src/components/chat-header/chat-header.component.jsx]: im not exactly sure why but when i put in `xs sm` the minimum distance which the grid will snap will decrease, thus i can push the elements closer together without them snapping to 2nd row. 

With `xs sm` i could achieve 340 px without it it snaps to second row at 380 px.

File: client/src/components/chat-header/chat-header.component.jsx (hunk 2 above)
Comment (Rocket-Fish) [reply client/src/components/chat-header/chat-header.component.jsx]: fixed in d095ea3

File: client/src/components/profile/profile.styles.jsx (hunk 3 above)
Comment (Rocket-Fish) [reply client/src/components/profile/profile.styles.jsx]: renamed to pad10 in d2b024c

**Pull request merged**